test/Microsoft.AspNetCore.Razor.Language.Test/Legacy/TagHelperBlockTest.cs

```diff
@@ -27,11 +27,20 @@ public void Clone_ClonesTagHelperChildren()
             var copy = (TagHelperBlock)tagHelper.Clone();
 
             // Assert
-            ParserTestBase.EvaluateParseTree(copy, tagHelper);
             Assert.Collection(
                 copy.Children,
                 child => Assert.NotSame(tagHelper.Children[0], child),
                 child => Assert.NotSame(tagHelper.Children[1], child));
+
+            if (GenerateBaselines)
```

Comment:
These are unit tests. They should not have been written this way to begin with 😞